Document the deprecated component file changes for 21.0 (#952)
Dyalog/documentation: language-reference-guide/docs/system-functions/query-external-variable.md

```

!!! Info "Information"

>     Support for external variables has been deprecated: they are read-only, it is not possible to create new ones, and support is scheduled for removal in a future release. For information on how to identify uses of external variables in your existing codebase, see the [Release Notes](../../../release-notes/announcements/deprecated-functionality/).

Reviewer: By default they are not supported at all (not read-only) but - for now - a switch allows their reinstatement,

>     Support for external variables has been deprecated: they are not supported by default, and support is scheduled for removal in a future release. Setting the [DYALOG_EXTVAR_SUPPORTED](../../../windows-installation-and-configuration-guide/configuration-parameters/dyalog-extvar-supported) configuration parameter to `1` reinstates support for the time being. For information on how to identify uses of external variables in your existing codebase, see the [Release Notes](../../../release-notes/announcements/deprecated-functionality/).

Reviewer 2: ```suggestion
    Support for external variables has been deprecated. They are no longer supported by default, although setting the [DYALOG_EXTVAR_SUPPORTED](../../../windows-installation-and-configuration-guide/configuration-parameters/dyalog-extvar-supported) configuration parameter to `1` reinstates support (support is scheduled for removal in a future release). For information on how to identify uses of external variables in your existing codebase, see the [Release Notes](../../../release-notes/announcements/deprecated-functionality/).
```
